android/app/src/main/res/values/styles.xml

```diff
@@ -6,6 +6,9 @@
         <item name="colorPrimary">@color/primaryColor</item>
         <item name="colorPrimaryDark">@color/primaryColorDark</item>
         <item name="android:textColor">#000000</item>
+        <!-- From `expo-splash-screen` setup instructions. -->
+        <item name="android:drawable">@color/splashscreen_background</item>
```

Comment:
I think this line isn't meant to be here: seems redundant with the next, and I don't see it in the linked instructions.

Reply:
Oh interesting; thanks for catching this. Looks a lot like what goes in `splashscreen.xml`, and I guess I might have thrown it in here too.